src/Test.FunctionalTests.Sarif/Multitool/ValidateCommandTests.cs

```diff
@@ -306,6 +307,16 @@ public void SARIF2015_EnquoteDynamicMessageContent_Invalid()
             => RunTest(MakeInvalidTestFileName(RuleId.EnquoteDynamicMessageContent, nameof(RuleId.EnquoteDynamicMessageContent)),
                 parameter: new TestParameters(verbose: true));
 
+        [Fact]
+        public void SARIF2016_FileUrisShouldBeRelative_Valid()
+            => RunTest(MakeValidTestFileName(RuleId.FileUrisShouldBeRelative, nameof(RuleId.FileUrisShouldBeRelative)),
+                parameter: new TestParameters(verbose: true, configFileName: "enable2016.configuration.xml"));
```

Comment:
>enable2016 [](start = 78, length = 10)

If the point is simply to enable a specific rule, we should have a helper that constructs the appropriate config using a rule id. And dispense with maintaining a checked in file.